modules/api-svc/.env.example

```diff
@@ -93,6 +93,17 @@ AUTO_ACCEPT_PARTY=false
 # confirmation call will be required to progress the requestToPay to transactionRequest state.
 AUTO_ACCEPT_R2P_PARTY=false
 
+# This parameter is only for the requestToPay transfers when the initiator is of type BUSINESS.
+# if set to false the SDK will not automatically accept quote on a requestToPayTransfer.
+# but will halt the operation after a quote response is received. A further
+# confirmation call will be required to progress the requestToPayTransfer to authorization step.
+AUTO_ACCEPT_R2P_BUSINESS_QUOTES=false
+
+# if set to false the SDK will not automatically accept OTP on a requestToPay operation
```

Comment:
I don't think anything needs to change right now, but I think we need to relook at this config to see if it can be more generic, and not specific to OTP.

Reply:
I didn't change anything. I just added those existing config params to example env file.